### PR TITLE
Removed bitcoinjs library and used crypto internal library

### PR DESCRIPTION
**Pull request by kelvinator07**

### CODE REVIEW

**Review by marcos-iov — DISMISSED** [review]
Well done! Share the PR with security for review before merging

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ console.log("Result: ", resultPmt);
 ```
 ### Tool Usage
 
-This library can be used via a tool that exists in the tool/pmt-builder.js file via the following command: `node tool/pmt-builder.js network blockHash txHash`, for example: `node tool/pmt-builder.js testnet 00000000000003d91235b675366fc6c26e0ea4b2f125fd292f164247d4e5b07e ac72bd61c72ac8143e5638998f479bdfc5834fce9576fa2054c7be93313abd66`
+This library can be used via a tool(this tool uses mempool.js api to get transactions) that exists in the tool/pmt-builder.js file via the following command: `node tool/pmt-builder.js network blockHash txHash`, for example: `node tool/pmt-builder.js testnet 00000000000003d91235b675366fc6c26e0ea4b2f125fd292f164247d4e5b07e ac72bd61c72ac8143e5638998f479bdfc5834fce9576fa2054c7be93313abd66`
```
Comment (nathanieliov) [inline README.md:49]:
```suggestion
This library can be used via a tool(this tool uses mempool.js api to get transactions) that exists in the tool/pmt-builder.js file via the following command: 

`node tool/pmt-builder.js network blockHash txHash`

 For example: 
 
 `node tool/pmt-builder.js testnet 00000000000003d91235b675366fc6c26e0ea4b2f125fd292f164247d4e5b07e ac72bd61c72ac8143e5638998f479bdfc5834fce9576fa2054c7be93313abd66`
```

What do you think about put commands in separate lines from the text? I prefer to see commands in that way. Top me that way it is easier to read.

File: index.js [hunk 2]
```diff
@@ -1,17 +1,19 @@
...
+
 const combineLeftAndRight = (left, right) => {
     let bufLeft = Buffer.from(left, 'hex');
     let bufRight = Buffer.from(right, 'hex');
     bufLeft.reverse();
     bufRight.reverse();
 
     let bufCombined = Buffer.concat([bufLeft,bufRight]);
-    let bufHashed = sha256(bufCombined);
+    let bufHashed = sha256(sha256(bufCombined));
```
Comment (nathanieliov) [inline index.js:16]: Nice!

**Review by nathanieliov — APPROVED** [review]
Well done K! LGTM, just left a small suggestion.

File: README.md (hunk 1 above)
Comment (kelvinator07) [reply README.md]: Yeah, Resolved

**Pull request merged**